Tee'sApp/Controllers/ViewControllerB.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  FirstViewController.swift
+//  Tee'sApp
+//
+//  Created by La Pommeray, Thalia on 10/27/22.
+//
+
+import UIKit
+
+class ViewControllerB: UIViewController {
+    
+    var info: String?;
+    let manager = InfoViewModel()
+    var userData:[User] = []
+    var user = User.dummyUser()
+    var userIndex = 0
+    var infoForAlert:String = ""
+    
+    @IBOutlet weak var userChoice: UILabel!
+    
+    @IBOutlet weak var userStreetAddress: UILabel!
+    
+    @IBOutlet weak var addressSuiteLabel: UILabel!
+    
+    @IBOutlet weak var addressCityLabel: UILabel!
+    @IBOutlet weak var userPhoneNumLabel: UILabel!
+    
+    @IBOutlet weak var userEmailLabel: UILabel!
+    override func viewDidLoad() {
+        super.viewDidLoad();
+        
+        // User info that will be displayed
+        userChoice.text = info
+        userStreetAddress.text? = ("\(user.address.street)")
+        addressSuiteLabel.text? = user.address.suite
+        var addy = "\(user.address.city) "
+        addy.append(user.address.zipcode)
+        addressCityLabel.text? = addy
+        userPhoneNumLabel.text? = user.phone
+        userEmailLabel.text? = user.email
```

Comment:
A lot of code inside the `viewDidLoad()` which is totally fine.  When I see all this outlet setting being done, maybe it would be a good idea to pull all of this out into a private function called `private func setupOutlets()` or something and put all this code into it.  That way, you have a single function call in your viewDidLoad and keeps things a bit cleaner in there